### PR TITLE
Update working on tctl nodes add usage

### PR DESCRIPTION
**Pull request by stevenGravy**
- fix misspelling
- token is not single-use as originally written

### CODE REVIEW

File: tool/tctl/common/usage.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ Examples:
   > tctl nodes add
 
   Generates a token when can be used to add a regular SSH node to the cluster.
-  The token genrated single-use token will be valid for 30 minutes.
+  The generated token will be valid for 30 minutes.
```
Comment (zmb3) [inline tool/tctl/common/usage.go:48]: While we're here I feel like we should replace this with `tctl tokens add`, since not everything is a "node".

Comment (stevenGravy) [reply tool/tctl/common/usage.go:48]: This is the description for `tctl help nodes add`. Do we want to reference you could use `tctl tokens add --type=node`?

Comment (zmb3) [reply tool/tctl/common/usage.go:48]: Ah, I see. Ignore me.

**Review by zmb3 — APPROVED** [review]

**Review by scottpgallagher — APPROVED** [review]
Approved.

**Pull request merged**